docs/LogGuide.md

```diff
@@ -2590,3 +2598,113 @@ Parsed Log Line Examples:
 ```
 
 <!-- AUTO-GENERATED-CONTENT:END (logLines:type=RSVData&lang=en-US) -->
+
+<a name="line263"></a>
+
+### Line 263 (0x107): StartsUsingExtra
+
+This line contains extra data for ActorCast/StartsUsing network data.
+
+This line is always output for a given StartsUsing cast. If the ability does not target the
```

Comment:
This is a belated thought for the OP patch, but I wonder if it'd be better if this line always printed the source position and then also included the target data if it existed (maybe with flags here too, if they exist). Mostly just saying that there's a lot of "figure out where things are when they start casting" that would benefit from always having the source actor position. (cc @xpdota too)

Reply:
Wouldn't it have the same issue as the native 20/21/22 lines, where a position read from memory might be stale?

Reply:
I think the hope was that it would have actor position data in the network packet as well, but I don't think that's the case?

Reply:
Yeah, that was my hope. I guess if it doesn't have it, then it doesn't have it.

Reply:
I think the only true solution for cast-before-move race conditions - without modifying the base plugin - is to hook ActorMove (and anything else that could move an actor), and have that data take priority over memory in 261-lines.